build/media_source/mod_community_info/css/community-info.css

```
@@ -0,0 +1,39 @@
+.select-location {
+  padding-left: 1em;
+  margin-bottom: .5rem;
+  font-weight: 400;
+}
+.text-right {
+  text-align: right;
```

```suggestion
  text-align: end;
```

Use logical css operators so that it works on RTL https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values/Floating_and_positioning

---

Done!